CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ All notable changes to this project will be documented in this file.
   - Abstract Barrier ``Barrier``
   - Frame Position Barrier ``PositionBarrier``
   - Body Spherical Barrier ``BodySphericalBarrier``
+- `ComTask` for Center of Mass tracking.
```

Comment:
All changes to the API should be documented in the changelog (so that users can track changes), therefore we should also add the new parameter (ignore/check limits) proposed in this PR.

Reply:
Ahh I see, will do next time!

Reply:
I have modified the CHANGELOG, with latest changes in API and restructured a bit